.gitattributes

```diff
@@ -0,0 +1 @@
+backends-velox/src/test/resources/tpch-approved-plan/** linguist-generated
```

Comment:
Nice job!

Reply:
But sadly by `linguist-generated` we can only fold the files in PR review view. It still doesn't reduce the changed line count. So this is still needed to be enhanced.